operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1004,16 +1023,16 @@ store_gateway:
   priorityClassName: null
 
   nodeSelector: {}
-  affinity:
-    podAntiAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchExpressions:
-              - key: target
-                operator: In
-                values:
-                  - store-gateway
-          topologyKey: 'kubernetes.io/hostname'
+  affinity: {}
+
+  # -- topologySpreadConstraints allows to customize part of the default topologySpreadConstraints. labelSelectors
+  # is generated by the chart using the same selectors as for services.
+  # It is recommended to replace this with requiredDuringSchedulingIgnoredDuringExecution podAntiAffinity rules when
```

Comment:
Let's add it to small, large, etc files like for the ingester then. Also I'm realizing that the anti affinity rule above will not work for new labels only for legacy , facepalm.

Reply:
Can I do this in a follow-up PR? This one is already quite old and has plenty of discussions.

Reply:
opened https://github.com/grafana/mimir/pull/2906